framework/src/mesh/MooseMesh.C

```diff
@@ -3232,7 +3230,7 @@ MooseMesh::buildFaceInfo() const
       if (!neighbor ||
           (neighbor->active() && (neighbor->level() == elem->level()) &&
            (elem_id < neighbor->id())) ||
-          (neighbor->level() < elem->level()))
+          (neighbor->active() && neighbor->level() < elem->level()))
```

Comment:
I don't think we should add this. If you want to add something, I would support adding a `mooseAssert` below. Something like
```
mooseAssert(neighbor->level() < elem->level() ? neighbor->active() : true, "If the neighbor is coarser than us, we expect that the neighbor must be active.");
```

Reply:
works for me